packages/renderers/lib/src/request/request_item_group_renderer.dart

```diff
@@ -1,11 +1,9 @@
-import 'package:collection/collection.dart';
 import 'package:enmeshed_types/enmeshed_types.dart';
 import 'package:flutter/material.dart';
+import 'package:renderers/renderers.dart';
```

Comment:
maybe like this:
```suggestion
import '/renderers.dart';
```
or leave it like the previous import:
```suggestion
import 'request_renderer_controller.dart';
import 'open_attribute_switcher_function.dart';
```